frontend/src/app/Navigation/index.jsx

```diff
@@ -74,47 +80,45 @@ function Sidebar({ collapsible }) {
             />
           )}
         </div>
-
-
-        <Menu mode="inline">
-          <Menu.Item key={'Dashboard'} icon={<DashboardOutlined />}>
+        <Menu mode="inline" selectedKeys={[currentPath]}>
+          <Menu.Item key={'/'} icon={<DashboardOutlined />}>
             <Link to={'/'} />
             Dashboard
           </Menu.Item>
-          <Menu.Item key={'Customer'} icon={<CustomerServiceOutlined />}>
+          <Menu.Item key={'/customer'} icon={<CustomerServiceOutlined />}>
```

Comment:
Instead of doing this, iterate the component and create a static array object constant that holds the `key` and the icon. so it will be easier to scale and reuse the component

Reply:
Ok changes done as requested, please review